src/features/health/agent.controller.js

```diff
@@ -103,5 +103,21 @@ const deleteAgent = async (req, res) => {
     }
 };
 
+// Method to get top agents by sales
+const getTopAgentsBySales = async () => {
+    try {
+        // Find agents with sales > 50000, sort by sales in descending order, and limit to top 3
+        const topAgents = await Agent.find({ sales: { $gt: 50000 } })
```

Comment:
This is a great solution, but I'm wondering why we're first filtering for sales over 50,000? What made you decide on this number?

Reply:
I entered the sales for each agent manually in the database and made sure the top agents had over $50000.

Reply:
Right, sounds good. But the question is more of about "why" instead of "how". The top three sales would be the top three regardless of what initial threshold you set, correct?